external-docs/docs/guides/configuration/qube-config/columns/attributes/index.md

```diff
@@ -0,0 +1,108 @@
+# Attribute columns
+
+This page discusses what an attribute column is, where one should be used, and how one can be defined.
+
+> For a detailed look at an attribute column's configuration options, see the [Reference table](#reference) at the
+bottom of this page.
+
+## What is an attribute column?
+
+Attribute columns describe or provide additional context to the data set's observed values. They **do not** identify
+a particular sub-set of the data set's population. If the column you are configuring does identify such a sub-set of
+the population, then it is probably a [dimension](../dimensions.md).
+
+## When to use one
+
+Unlike [dimensions](../dimensions.md), [observations](../observations.md), [measures](../measures.md) and
+[units](../units.md), attributes are optional components of a data cube. Their primary usage is to qualify
+observed values by providing additional context about individual data points. Two common examples of attributes are
+**observation status** and **confidence intervals**.
+
+| Year | Location | Value |                  Measure |                   Unit |      Status | 95% CI (lower bound) | 95% CI (upper bound) |
```

Comment:
If you change these examples to use the pivoted shape instead of the standard shape it'll narrow these tables quite substantially and give us a higher signal/noise ratio here. This would be good to do in all the attribute sections.